src/vs/workbench/contrib/debug/browser/welcomeView.ts

```diff
@@ -107,37 +107,75 @@ export class WelcomeView extends ViewPane {
 
 const viewsRegistry = Registry.as<IViewsRegistry>(Extensions.ViewsRegistry);
 viewsRegistry.registerViewWelcomeContent(WelcomeView.ID, {
-	content: localize({ key: 'openAFileWhichCanBeDebugged', comment: ['Please do not translate the word "commmand", it is part of our internal syntax which must not change'] },
-		"[Open a file](command:{0}) which can be debugged or run.", (isMacintosh && !isWeb) ? OpenFileFolderAction.ID : OpenFileAction.ID),
+	content: localize(
+		{
+			key: 'openAFileWhichCanBeDebugged',
+			comment: [
+				'Please do not translate the word "command", it is part of our internal syntax which must not change',
+				'{Locked="](command:{0})"}'
+			]
+		},
+		"[Open a file](command:{0}) which can be debugged or run.", (isMacintosh && !isWeb) ? OpenFileFolderAction.ID : OpenFileAction.ID
```

Comment:
It's a little more complicated, but the better way to do this imo is something like this:

```ts
localize('', '{0} which can be debugged or run', `[${localize('', 'Open a file')}](command:${(isMacintosh && !isWeb) ? OpenFileFolderAction.ID : OpenFileAction.ID})`)
```

That prevents the translator potentially messing up.

Reply:
I disagree. The sentence is `Open a file which can be debugged or run`. Breaking it up could mean that `Open a file` gets translated in a weird way that doesn't flow with `which can be debugged or run`.

Reply:
Example:
![image](https://user-images.githubusercontent.com/2644648/215170567-580928cc-d058-4427-b288-fd7fa24e90b6.png)
vs
![image](https://user-images.githubusercontent.com/2644648/215170711-5fec22d1-5604-4ea5-8909-a7517851f81d.png)

Reply:
That's true. How about putting markers instead and explaining in the command it needs to wrap the text?

```ts
"{0}Open a file{1} which can be debugged or run."
```

Reply:
That would also work.